docs/community/working-group-definitions.md

```diff
@@ -0,0 +1,198 @@
+# Working Group Definitions
+
+This document maintains a list of the current PowerShell [Working Groups (WG)](working-group.md),
+as well as their definitions, membership, and a non-exhaustive set of examples of topics that fall
+within the purview of that WG.
```

Comment:
@joeyaiello I think we should have @iSazonov included in some of the WGs. He is a maintainer and, more importantly, he has been doing the WG work for the PowerShell repo for a long time already.

Reply:
@iSazonov can you please comment on what WGs you'd like to be part of?

Reply:
I think we don't need to mix maintenance and expertise.
Today I do really full pre-moderation all issues and PR so that MSFT team gets clean issues and clean code and tests in PRs.
For issues the pre-moderation is I set right area/WG label and adjust a level label (I usually use `Issue-Question` as an entry point because the automatically set label does not match the content of the request.)

I would be happy if the working groups did this work. :-) 
If not. I can continue the pre-moderation leaving `Needs-Triage` label to the working groups.

_**Today I suggest to set `Needs-Triage` label to all open issues by a script.**_

Reply:
You participated in a lot discussions in various issues/PRs and also did a lot work in many aspects of PowerShell already, so I don't doubt you have certain level of expertise in some areas of PowerShell that you feel comforatble with. But of course, it's not mandatory for mantianer to join any WGs, so it's totally up to you :)

The `Needs-Triage` label should have been applied to new in-coming issues since I have merged the updates to the issue template.

Reply:
I don't mind participating in working groups if they include me.

> Do you notice any new issue not marked with needs-triage label?

No, this works. I mean we could use a script and assign the label to all old issues - today it is not clear whether they was already triaged or no.